environment.yml

```diff
@@ -3,10 +3,13 @@ channels:
   - conda-forge
```

Comment:
i'm curious @mbjoseph does this envt need to be in this repo? i have a full version of the envt that i maintain elsewhere is why i'm asking but that envt doesn't have pytest in it as the students don't need it. thoughts?

Reply:
Good question.... I think you would want to have some kind of file that specifies the requirements for the earthpy package in this repo. This doesn't necessarily need to be identical to the environment that you use for teaching though. 

I would guess that the minimal set of requirements for earthpy is a subset of all of the packages you would want to use in your class, right? e.g., you want to use Jupyter Lab to teach with, but it's not necessary for earthpy to run. So, I wonder whether there is a way to have a requirements file for earthpy that lives on its GitHub repo, and then a separate environment.yml file for your class?

Reply:
@lwasser maybe we can open another issue to figure out where then evironment.yml file should live? Seems to be a separate problem than adding Travis CI + Codecov to the repo. We can change the environment.yml file later on and adjust Travis CI and Codecov to match our solution later on once we reach a decision.